### DISCUSSION TITLE
Optional string enumValue?

### DISCUSSION THREAD

**Discussion by rickpasetto** [firecmsco/firecms#390] [Q&A] [answered]
First, let me say this is an awesome framework.  Thanks to everyone who has contributed!
I'm still fairly new to this library, so I'm still learning how to use it.  I'm trying to do something I think is fairly simple: add a field to a collection that you can select from an enum, just like the example in the docs, e.g.:

```
const colors = {
   RED: "red",
   GREEN: "green",
   BLUE: "blue"
} 
...
{
    name: "Color",
    dataType: "string",
    validation: { required: false },
    enumValues: colors
}
```

However, notice that I want this to be an optional field, so can be removed from the Firebase object.  I want the user to be able to "un-set" the field in the UI.  When I run this, the field initially shows as blank, but after it is set, there's no way to "delete" the field value.

Is there a way to do this?

Any help would be greatly appreciated.

**Comment by fgatti675** [comment]
Hi @rickpasetto 
This is not possible at the moment.
As crazy as this sounds, this issue has not been raised in the 2 years life of the project 😂
Feel free to create a new issue! It should be fairly straightforward to implement

**Reply by fgatti675** [reply]
It is possible now with the `clearable` prop in some properties

**Comment by fgatti675** [comment, marked as answer]
It is possible now with the `clearable` prop in some properties

**Comment by serhii-sierov** [comment]
Used `clearable: true`. Got a dropdown with a “Clear” icon. But when clicking “Clear”, it doesn’t clear the value — it just opens the dropdown.
(Relevant for string properties where enumValues are set)

**Comment by fgatti675** [comment]
Thanks for reporting! Looking into it now

**Comment by serhii-sierov** [comment]
It seems it is a known issue: https://github.com/firecmsco/firecms/issues/391
